.vib/external-dns/goss/goss.yaml

```diff
@@ -0,0 +1,27 @@
+port:
+  tcp6:7979:
```

Comment:
I would have done this using templating but this port is not stated as an environment variable.